Mon2Cam.sh

```diff
@@ -149,22 +188,29 @@ then
 	done
 fi
 
-# Monitor information
-MONITOR_INFO=$(xrandr --listactivemonitors | grep "$MONITOR_NUMBER:" | cut -f4 -d' ')
-MONITOR_HEIGHT=$(echo "$MONITOR_INFO" | cut -f2 -d'/' | cut -f2 -d'x')
-MONITOR_WIDTH=$(echo "$MONITOR_INFO" | cut -f1 -d'/')
-MONITOR_X=$(echo "$MONITOR_INFO" | cut -f2 -d'+')
-MONITOR_Y=$(echo "$MONITOR_INFO" | cut -f3 -d'+')
-
-# Use x11grab to stream screen into v4l2loopback device
-echo "CTRL + C to stop"
-echo "Your screen will look mirrored for you, not others"
-$FFMPEG \
-	-f x11grab \
-	-r "$FPS" \
-	-s "$MONITOR_WIDTH"x"$MONITOR_HEIGHT" \
-	-i "$DISPLAY"+"$MONITOR_X","$MONITOR_Y" \
-	$FFMPEG_OPTIONS \
-	-pix_fmt yuv420p \
-	-f v4l2 \
-	/dev/video"$DEVICE_NUMBER" &> /dev/null
\ No newline at end of file
+	# Use x11grab to stream screen into v4l2loopback device
+	echo "CTRL + C to stop"
+	echo "Your screen will look mirrored for you, not others"
```

Comment:
I think the indentation is wrong here.

Reply:
Fixed in 88921f42685b5046224bbe6771edd377275d6c27.